doc/authentication.md

```diff
@@ -108,20 +108,39 @@ Alternatively, the clients can provide the token directly when adding the remote
 
 In a {abbr}`PKI (Public key infrastructure)` setup, a system administrator manages a central PKI that issues client certificates for all the LXD clients and server certificates for all the LXD daemons.
 
-In PKI mode, TLS authentication requires that client certificates are signed be the CA.
+In PKI mode, TLS authentication requires that client certificates are signed be the {abbr}`CA (Certificate authority)`.
 This requirement does not apply to clients that authenticate via [OIDC](authentication-openid).
 
-To enable PKI mode, complete the following steps:
+The steps for enabling PKI mode differ slightly depending on whether you use an ACME provider in addition (see {ref}`authentication-server-certificate`).
 
-1. Add the {abbr}`CA (Certificate authority)` certificate to all machines:
+`````{tabs}
+````{group-tab} Only PKI
+If you use a PKI system, both the server certificates and the client certificates are issued by a secondary CA.
+
+1. Add the CA certificate to all machines:
    - Place the `client.ca` file in the clients' configuration directories (`~/.config/lxc` or `~/snap/lxd/common/config` for snap users).
    - Place the `server.ca` file in the server's configuration directory (`/var/lib/lxd` or `/var/snap/lxd/common/lxd` for snap users).
```

Comment:
In a cluster setup, would this file be called `cluster.ca`? Or is the current description valid for both single server and cluster setup?

Reply:
This description is only valid for standalone LXD. For clustered LXD, it should be `cluster.ca` (plus `cluster.crt` and `cluster.key` in the lines below) and all cluster members must have the same certificates.

Also, while the `{server,cluster}.ca` must be present before LXD starts, the `{server,cluster}.{crt,key}` files can be updated at runtime using `lxc cluster update-certificate`.